config/finders/citizen_readiness_email_signup.yml

```diff
@@ -0,0 +1,19 @@
+---
+base_path: "/prepare-eu-exit/all/email-signup"
+content_id: 6a46087b-737e-40c8-bcf2-afa580b7263e
+description: You'll get an email each time content is published.
+document_type: finder_email_signup
+title: Detailed EU Exit guidance
+details:
+  filter:
+    all_part_of_taxonomy_tree: d7bdaee2-8ea5-460e-b00d-6e9382eb6b61
+  email_filter_by:
+  email_filter_name:
+  subscription_list_title_prefix: 'Detailed EU Exit guidance '
+  email_filter_facets:
+  - facet_id: topics
```

Comment:
Will the `facet_id` work, if given `any_part_of_taxonomy_tree` as the `key` by the finder?

Reply:
Thanks, I've fixed this